js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -2801,6 +2802,7 @@ eventTypes.OAUTH2_DEVICE_CODE_TO_TOKEN.name=OAuth2 device code to token
 searchGroups=Search groups
 trusted-hosts.tooltip=List of Hosts, which are trusted and are allowed to invoke Client Registration Service and/or be used as values of Client URIs. You can use hostnames or IP addresses. If you use star at the beginning (for example '*.example.com' ) then whole domain example.com will be trusted.
 disableNonceHelp=Do not send the nonce parameter in the authentication request. The nonce parameter is sent and verified by default.
+disableTypeClaimCheckHelp=Do not validate the type claim. The type claim is validated by default.
```

Comment:
I think it would be helpful to enhance this message a bit for clarity, e.g.:
```
Disables the validation of the `typ` claim of the received tokens. If this is `off` the type claim is validated (default).
```

Reply:
done